gradle/libs.versions.toml

```diff
@@ -22,6 +22,7 @@ lifecycle = "2.7.0"
 activity = "1.9.1"
 media3 = "1.4.0"
 camera = "1.3.4"
+emoji2 = "1.5.0-beta01"
```

Comment:
Note for the reviewer- I'm not sure if non-release versions are acceptable, but given that this library hasn't had a release in over a year, and this newer build adds support for the newer Unicode 15.1 emojis I figured it might be okay